packages/lesswrong/lib/collections/posts/collection.ts

```diff
@@ -68,7 +69,8 @@ makeEditable({
     pingbacks: true,
     permissions: {
       viewableBy: ['guests'],
-      editableBy: ['members', 'sunshineRegiment', 'admins'],
+      // TODO: do we also need to cover userIsPostGroupOrganizer somehow?
```

Comment:
Yes I think you do. I claim you should do this by letting userIsPostGroupOrganizer edit the post metadata and include it in the function.

Reply:
Reading the functions, currently canUserEditPostMetadata already tries to check if the user is a group organizer by getting it from `group.organizerIds`, whereas `userIsPostGroupOrganizer` checks it by running a query against Localgroups.

Reply:
I think the issue here is that `group.organizerIds` only exists on posts which were returned via a gql resolver (`groupId` is a foreign key field that adds a `group` resolver).  So on the server side, an organizer might actually not be allowed to edit an event's contents even if they're an organizer, though I'll see if I can figure out how to test that.

Reply:
It turns out that the current behavior on `master` already doesn't let group co-organizers edit events unless they authored the event post themselves, so this won't be a regression.